src/styledComponents/chips/OutlinedChip.tsx

```diff
@@ -0,0 +1,84 @@
+import type { ChipProps } from '@mui/material';
+import type { CSSProperties } from 'react';
+
+import { Chip, styled } from '@mui/material';
+
+import {
+    defaultOutlineColor,
+    defaultOutlineColor_hovered,
+    diminishedTextColor,
+} from 'src/context/Theme';
+
+export const OutlinedChip = styled(Chip, {
+    // TODO (typing): Consider creating a typed, utility function for property forwarding
+    //   that is able to handle an array of properties.
+    shouldForwardProp: (props) => props !== 'diminishedText',
+})<
+    ChipProps & {
+        diminishedText?: boolean;
+    }
+>(({ color, diminishedText, style, theme }) => {
+    const colorKey = color ?? 'default';
+
+    const background =
+        colorKey === 'default'
+            ? theme.palette.background.default
+            : theme.palette[colorKey].alpha_12;
+
+    const borderStyle: CSSProperties['borderStyle'] = diminishedText
+        ? 'dashed'
+        : 'solid';
+
+    const border =
+        colorKey === 'default'
+            ? `1px ${borderStyle} ${defaultOutlineColor[theme.palette.mode]}`
+            : `1px ${borderStyle} ${theme.palette[colorKey].alpha_50}`;
```

Comment:
Do you foresee the borders differing more? If not I think it might be safer to just put the ternary in where the color is set. That way we can always keep the other settings identical.